modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/form/validation/util/DateParameterUtilTest.java

```diff
@@ -62,6 +62,29 @@ public void testGetLocalDateTime() {
 		Assert.assertNull(DateParameterUtil.getLocalDateTime(StringPool.BLANK));
 	}
 
+	@Test
+	public void testGetLocalDateTime2() {
+		Assert.assertEquals(
+			"2021-10-28T00:00",
+			String.valueOf(DateParameterUtil.getLocalDateTime("2021-10-28")));
+
+		Assert.assertEquals(
+			"2021-10-28T01:00",
+			String.valueOf(
+				DateParameterUtil.getLocalDateTime("2021-10-28 01:00:00")));
+
+		Assert.assertEquals(
+			"2021-10-28T01:00",
+			String.valueOf(
+				DateParameterUtil.getLocalDateTime("2021-10-28 1:00")));
+
+		Assert.assertEquals(
+			"2024-02-06T14:43:44",
+			String.valueOf(
+				DateParameterUtil.getLocalDateTime(
+					"Tue Feb 06 14:43:44 GMT 2024")));
+	}
```

Comment:
I think its better to add those cases to the `testGetLocalDateTime` method, without creating a new one.